Adding references about the Qiskit Code Assistant service is available only through the IBM Quantum Platform classic (#3231)
Qiskit/documentation: docs/guides/qiskit-code-assistant-vscode.mdx

Learn how to install, use, configure and uninstall the official Qiskit Code Assistant extension in Visual Studio Code (VS Code).

<Admonition type="note" title="Notes">

>     - This is an experimental feature available only to IBM Quantum&reg; Premium Plan users registered in the [IBM Quantum Platform Classic](https://quantum.ibm.com/).

Reviewer: ```suggestion
    - This is an experimental feature available only to IBM Quantum&reg; Premium Plan users with[IBM Quantum Platform Classic](https://quantum.ibm.com/) accounts.
```

- To change keyboard shortcuts, open the Keyboard Shortcuts settings (`Cmd/Ctrl`+`Shift`+`P` -> `Preferences: Open Keyboard Shortcuts (JSON)`) and search for `qiskit-vscode`.

> - You can change the IBM Quantum API key to use in the VS Code command palette. To do that, type `Cmd/Ctrl`+`Shift`+`P`, search for `qiskit`, select the **Qiskit Code Assistant: Set IBM Quantum API token** command, and paste your IBM Quantum Classic API key.

Reviewer: ```suggestion
- You can change the IBM Quantum Classic API token to use in the VS Code command palette. To do that, type `Cmd/Ctrl`+`Shift`+`P`, search for `qiskit`, select the **Qiskit Code Assistant: Set IBM Quantum API token** command, and paste your IBM Quantum Classic API token.
```
